### PR TITLE
Try to improve support for py32f002a

### PR DESCRIPTION
**Pull request by usbalbin**
This is not yet tested on real hardware. So for this is just an attempt to get the code to compile with the device feature `py32f002a`.

### CODE REVIEW

File: src/serial.rs [hunk 1]
```diff
@@ -67,7 +67,10 @@ use core::{
 use embedded_hal::prelude::*;
 
 use crate::{rcc::Rcc, time::Bps};
-use crate::gpio::{gpioa::*, gpiob::*, gpiof::*, Alternate, AF0, AF1, AF3, AF4, AF8, AF9};
+use crate::gpio::{gpioa::*, gpiob::*, gpiof::*, Alternate, AF0, AF1, AF8};
+
+#[cfg(any(feature = "py32f030", feature = "py32f003"))]
+use crate::gpio::{AF3, AF4, AF8, AF9};
```
Comment (creatoy) [inline src/serial.rs:73]: AF8 reimported here

**Review by creatoy — CHANGES_REQUESTED** [review]
Thanks for your contrib. I'm not have a real py32f002a dev board now, So I can not test it as well. It would be even better if you could test it on a real chip.

**Comment by usbalbin** [comment]
This now runs successfully(but with some warnings)
```bash
cargo c --features py32f030 && cargo c --features py32f003 && cargo c --features py32f002a
```

File: src/serial.rs (hunk 1 above)
Comment (usbalbin) [reply src/serial.rs]: Oh sorry, fixed

**Review by creatoy — APPROVED** [review]

**Pull request merged**